### PR TITLE
Made site work with latest hugo version

### PR DESCRIPTION
**Pull request by kolaente**
As title. I didn't update everything inside of `content/pages` as it seem to not be used anymore.

### CODE REVIEW

File: content/cross-platform/index.en-us.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: "Cross-platform"
 weight: 10
 toc: false
 draft: false
+url: "en-us"
```
Comment (techknowlogick) [inline content/cross-platform/index.en-us.md:7]: Does capitalization matter? As when I go to https://gitea.io I get redirected to https://gitea.io/en-US/

**Review by tboerger — APPROVED** [review]

File: content/cross-platform/index.en-us.md (hunk 1 above)
Comment (kolaente) [reply content/cross-platform/index.en-us.md:7]: Seems like it doesn't matter, the links in the footer to localized version are all lowercase.

**Comment by techknowlogick** [comment]
LGTM

File: content/cross-platform/index.en-us.md (hunk 1 above)
Comment (techknowlogick) [reply content/cross-platform/index.en-us.md:7]: Sounds good! Thanks for quick response.

Comment (kolaente) [reply content/cross-platform/index.en-us.md:7]: Well I thank you for that quick LGTM :)

**Pull request merged**